phoenix/phoenix_resource.ini

```diff
@@ -48,6 +48,7 @@ GENOME_SUBVERSION_NAME=hg38tgen
 SAMTOOLS_VERSION="1.9"
 
 REFERENCE_DNA_BASE_DOWNLOAD=ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/001/405/GCA_000001405.15_GRCh38/seqs_for_alignment_pipelines.ucsc_ids/GCA_000001405.15_GRCh38_full_plus_hs38d1_analysis_set.fna.gz
+REFERENCE_DNA_GENOME_BASENAME=/home/tgenref/homo_sapiens/grch38_hg38/hg38tgen/genome_reference/GRCh38tgen_decoy_alts_hla
```

Comment:
@bryce-turner this looks like a variable that could be created on the fly as opposed to requiring the user to define, its not an external file or required tool

Reply:
Very true, this has been resolved in commit 9565f8b.